packages/router/src/url_tree.ts

```diff
@@ -13,65 +13,149 @@ export function createEmptyUrlTree() {
   return new UrlTree(new UrlSegmentGroup([], {}), {}, null);
 }
 
-export function containsTree(container: UrlTree, containee: UrlTree, exact: boolean): boolean {
-  if (exact) {
-    return equalQueryParams(container.queryParams, containee.queryParams) &&
-        equalSegmentGroups(container.root, containee.root);
-  }
+/**
+ * A set of options which specify how to determine if a `UrlTree` is active, given the `UrlTree`
+ * for the current router state.
+ *
+ * @publicApi
+ * @see Router.isActive
+ */
+export interface IsActiveUrlTreeOptions {
```

Comment:
Consider the following rename to be consistent with the param name and also internal names like `subsetMatchOptions`, etc. Also, these options apply to url strings, not just UrlTrees.

```suggestion
export interface IsActiveMatchOptions {
```